### PR TITLE
Dawn/network protos

### PR DESCRIPTION
**Pull request by jianahuang10**
Fixed proto definition to make the header an enum.

### CODE REVIEW

**Review by kevinmasd — DISMISSED** [review]
Seems to have a merge conflict. Also, has runtime been notified of these changes?

File: ansible-protos/notification.proto [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+message Notification {
+	enum Type {
+	    CONSOLE_LOGGING = 0;
+	    STUDENT_SENT = 1;
+	    STUDENT_RECEIVED = 2;
+	    STUDENT_NOT_RECEIVED = 3;
+	}
+    Type header = 1;
+    string console_output = 2;   // Console Output From Runtime to Dawn
+}
```
Comment (yizow) [inline ansible-protos/notification.proto]: Newline at end of file

**Review by yizow — CHANGES_REQUESTED** [review]
Style

File: ansible-protos/notification.proto (hunk 1 above)
Comment (jianahuang10) [reply ansible-protos/notification.proto]: Fixed

**Comment by jianahuang10** [comment]
Talked to @bodiddles about the change and fixed merge conflict.

**Review by yizow — APPROVED** [review]

**Pull request merged**